content/docs/runbooks/admin/create-operator.md

```diff
@@ -0,0 +1,109 @@
+
+# How to Create an Operator from Scratch
+This is a step-by-step walk through of the creation of an operator using the KUDO cli to generate the operator structure.
+
+## Preconditions
+
+None
+
+## Steps
+
+### Create the Core Operator Structure
+
+```
+# create operator folder
+mkdir first-operator
+cd first-operator
+kubectl kudo package new first-operator
+```
+
+This creates the main structure of the operator which can be view with `tree`
+
+```
+tree .
+.
+└── operator
+    ├── operator.yaml
+    └── params.yaml
+```
```

Comment:
I don't think `tree` is available by default on MacOs, not sure about linux. Maybe rather use `ls -aR` or something similar?